keyboards/paladintkl/keymaps/default/keymap.c

```diff
@@ -0,0 +1,105 @@
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM fn_actions[] = {};
```

Comment:
Could you remove the `fn_actions` block?  All of it's features have been added as keycodes, and been deprecated in favor of `process_record_user.